src/components/layerpublisher/LayerPublisher.js

```diff
@@ -125,7 +128,9 @@ export default class LayerPublisher extends React.Component {
       const waitingOn = Object.entries(prevState.pleaseWaits)
         .filter(entry => entry[1])
         .map(entry => entry[0]);
-      console.warn(`cleared "${layerName}"; waiting on ${waitingOn.length}:`, waitingOn);
+      if (waitingOn.length > 0) {
+        console.warn(`cleared "${layerName}"; waiting on ${waitingOn.length}:`, waitingOn);
+      }
```

Comment:
It is more log noise, but I think it can be helpful to see the empty lists in the log, so you know that everything has in fact loaded.

Reply:
The only thing is that it fires on all pubsub events, of which there are a lot with the sliders.  I am open to other suggestions but I think it actually slows down the application (maybe I'm being paranoid).